rotkehlchen/chain/evm/frontend_structures/events_mappings.py

```diff
@@ -0,0 +1,170 @@
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.chain.evm.frontend_structures.types import EventDetails, TransactionEventType
```

Comment:
This approach is wrong.

Each decoder should add to these mappings on its own and those should be added dynamically as decoders are created. Much like counterparties and accounting rules are created.

We are creating a modular system not a hard-coded mapping ...